bs/src/tablecloth.ml

```diff
@@ -740,7 +886,7 @@ module StrSet = struct
 
   let to_list = toList
 
-  let ofList (s : value list) : t = s |> Array.of_list |> Set.fromArray
+  let ofList (s : value list) : t = s |> Caml.Array.of_list |> Set.fromArray
 
```

Comment:
Belt.List.toArray? Same below

Reply:
@j-m-hoffmann did you know about GitHub's [suggestion](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) feature? Its perfect for one-liner changes like these since I can incorporate your improvements directly in the GitHub UI.

Reply:
thnx for the tip, will do so next time